### PR TITLE
changed the logic for id_token to contain PS as _vp_token

### PR DESCRIPTION
**Pull request by sksadjad**
- contains changes in constructing id_token and vp_token (and how we process them using pex)
- also changed the ethr dids that were failing to resolve

### CODE REVIEW

File: test/PresentationExchange.spec.ts [hunk 1]
```diff
@@ -370,7 +370,7 @@ describe('presentation exchange manager tests', () => {
       format: VerifiablePresentationTypeFormat.LDP_VP,
     };
     try {
-      await PresentationExchange.validatePayloadsAgainstDefinitions(pd, [vpw]);
+      await PresentationExchange.validatePayloadsAgainstDefinitions(pd, [vpw], undefined);
```
Comment (nklomp) [inline test/PresentationExchange.spec.ts:373]: Why do we need the undefined here?

File: src/main/AuthenticationResponse.ts [hunk 2]
```diff
@@ -288,14 +289,57 @@ async function createSIOPResponsePayload(
...
+      iss: ResponseIss.SELF_ISSUED_V2,
+      aud: verifiedJwt.payload.redirect_uri,
+      iat: Date.now() / 1000,
+      exp: Date.now() / 1000 + (resOpts.expiresIn || 600),
+      sub: resOpts.did,
+      //TODO: We don't have this value in the payload
+      auth_time: verifiedJwt.payload.auth_time,
+      nonce: nonce,
+      _vp_token: {
+        // right now pex doesn't support presentation_submission for multiple VPs (as well as multiple VPs)
```
Comment (nklomp) [inline src/main/AuthenticationResponse.ts:304]: Can we create a ticket that we refine later in Jira and link to it

**Review by nklomp — COMMENTED** [review]
Some small remarks

File: src/main/AuthenticationResponse.ts (hunk 2 above)
Comment (sksadjad) [reply src/main/AuthenticationResponse.ts:305]: created 3:
1. to support `presentation_submission` outside of VP in PEX
2. to support multiple VPs in PEX
3. to handle multiple VPs in this project (did-auth-siop)

File: test/PresentationExchange.spec.ts (hunk 1 above)
Comment (sksadjad) [reply test/PresentationExchange.spec.ts]: made it optional

**Pull request merged**